docs/howtoguides/use_https_in_https_proxy.rst

```diff
@@ -0,0 +1,65 @@
+How to use an HTTPS proxy that uses HTTPS itself (HTTPS-in-HTTPS)
+*****************************************************************
+
+.. note:: Support for HTTPS-in-HTTPS proxying was added in version 28.1
+
+.. warning:: HTTPS-in-HTTPS is only supported by the Ubuntu Pro Client on Ubuntu 18.04 and later
+
+If you need to set ``https_proxy`` to a proxy that uses ``https://``, then you need to follow a couple extra steps to ensure all Ubuntu Pro actions work correctly.
+
+
+Install extra pycurl dependency
+===============================
+
+``pro`` switches to use ``pycurl`` only for HTTPS-in-HTTPS proxying, but ``pycurl`` may not be installed by default on your machine. To install it, run:
+
+.. code:: bash
+
+   sudo apt install python3-pycurl
+
+.. note::
+   If you don't do this and try to set the HTTPS-in-HTTPS proxy anyway, you will get an error that looks like this:
+
+   .. code:: none
+
+      To use an HTTPS proxy for HTTPS connections, please install pycurl with `apt install python3-pycurl`
+
+Ensure the proxy's HTTPS certificate will be trusted by Livepatch
+=================================================================
+
+.. tip:: You can skip this step if you don't plan on using Livepatch
+
+Livepatch requires configuration separate from the rest of the Ubuntu system for trusting certificates. Even if your proxy's certificate is signed by a well known CA, it may not be trusted by livepatch by default.
+
+To ensure livepatch will trust your proxy's certificate, first pre-install the livepatch-client:
+
+.. code:: bash
+
+   sudo snap install canonical-livepatch
+
+Then download the certificate of your proxy (or of the CA) in PEM format and configure livepatch-client to trust it:
```

Comment:
It's usually the default format, but just in case someone ends up with a cert in non-PEM, I included this detail here. https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail has a good high level description and links to the IETF RFC that formalizes the format, I'll add a link to that